### DISCUSSION TITLE
Does it support tesseract 5? steve8000818@gmail.com

### DISCUSSION THREAD

**Discussion by nissansz** [buliasz/tesstrain-windows-gui#12] [Q&A] [answered]
Does it support tesseract 5? steve8000818@gmail.com

**Comment by buliasz** [comment, marked as answer]
Definitely. It is tested on version 5 but should work with all 4+ versions as well.

**Comment by nissansz** [comment]
Do you have trained Chinese / Japanese models? How about accuracy vs paddleocr (crnn)?

Below is my training result on scene text for Japanese drama.
![image](https://user-images.githubusercontent.com/41010669/173844331-cafac7b7-fb20-4288-bd29-f556bd02fae1.png)

**Comment by nissansz** [comment]
I tried your sample data. It works to train. But how to prepare Japanese dataset and how to configure settings for Japanese in Gui?

Warning: given outputs 77 not equal to unicharset of 76.
Num outputs,weights in Series:
  1,36,0,1:1, 0
Num outputs,weights in Series:
  C3,3:9, 0
  Ft16:16, 160
Total weights = 160
  [C3,3Ft16]:16, 160
  Mp3,3:16, 0
  TxyLfys48:48, 12480
  Lfx96:96, 55680
  RxLrx96:96, 74112
  Lfx192:192, 221952
  Fc76:76, 14668
Total weights = 379052
Built network:[1,36,0,1[C3,3Ft16]Mp3,3TxyLfys48Lfx96RxLrx96Lfx192Fc76] from request [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx192 O1c77]
Training parameters:
  Debug interval = 0, weights = 0.1, learning rate = 0.002, momentum=0.5
null char=75
2 Percent improvement time=100, best error was 100 @ 0
At iteration 100/100/100, Mean rms=6.824000%, delta=43.526000%, BCER train=99.900000%, BWER train=100.000000%, skip ratio=0.000000%,  New best BCER = 99.900000 wrote checkpoint.

At iteration 200/200/200, Mean rms=6.729000%, delta=41.808000%, BCER train=99.941000%, BWER train=100.000000%, skip ratio=0.000000%,  New worst BCER = 99.941000 wrote checkpoint.

At iteration 300/300/300, Mean rms=6.689000%, delta=41.587000%, BCER train=99.961000%, BWER train=100.000000%, skip ratio=0.000000%,  New worst BCER = 99.961000 wrote checkpoint.

2 Percent improvement time=400, best error was 100 @ 0
At iteration 400/400/400, Mean rms=6.662000%, delta=40.956000%, BCER train=99.856000%, BWER train=100.000000%, skip ratio=0.000000%,  New best BCER = 99.856000 wrote checkpoint.

2 Percent improvement time=500, best error was 100 @ 0
At iteration 500/500/500, Mean rms=6.630000%, delta=40.401000%, BCER train=99.046000%, BWER train=100.000000%, skip ratio=0.000000%,  New best BCER = 99.046000 wrote checkpoint.

2 Percent improvement time=600, best error was 100 @ 0
At iteration 600/600/600, Mean rms=6.605000%, delta=39.960000%, BCER train=98.374000%, BWER train=100.000000%, skip ratio=0.000000%,  New best BCER = 98.374000 wrote checkpoint.

2 Percent improvement time=300, best error was 99.856 @ 400
At iteration 700/700/700, Mean rms=6.582000%, delta=39.464000%, BCER train=97.234000%, BWER train=100.000000%, skip ratio=0.000000%,  New best BCER = 97.234000 wrote checkpoint.

2 Percent improvement time=300, best error was 99.046 @ 500
At iteration 800/800/800, Mean rms=6.568000%, delta=39.257000%, BCER train=96.503000%, BWER train=100.000000%, skip ratio=0.000000%,  New best BCER = 96.503000 wrote checkpoint.

**Comment by nissansz** [comment]
I tried 100 lines jp. It can be trained. Result on training data is fine, either as new model and refined model.

**Comment by buliasz** [comment]
I'm glad you've managed to do it.
Just FYI there's a "Discussions" section which is more appropriate for such questions that the "Issues" section.
